controller/src/controller/models/domain/rsptag.py

```diff
@@ -1,6 +1,7 @@
 """Abstract data types for handling RSP image tags."""
 
 import contextlib
+import datetime
```

Comment:
```suggestion
from datetime import UTC, datetime
```
This is what we do most other places in our code bases.